frontend/src/metabase/querying/hooks/use-date-filter/use-date-filter.unit.spec.ts

```diff
@@ -125,11 +125,13 @@ function getTestCases(
         type: "specific",
         operator: "=",
         values: [date1],
+        hasTime: false,
```

Comment:
This file could use tests with `hasTime: true`.

When I use "After" (or "Before") filter with time, the filter has display name `Created At: Minute is after Dec 12, 2024, 12:00 AM`
But I would expect it to be `Created At: is after Dec 12, 2024, 12:00 AM`

This looks unrelated to this PR though.